configure.php

```diff
@@ -109,7 +109,11 @@ function remove_prefix(string $prefix, string $content): string {
     exit(1);
 }
 
-$files = explode(PHP_EOL, run('grep -E -r -l -i ":author|:vendor|:package|VendorName|skeleton|vendor_name|vendor_slug|author@domain.com" --exclude-dir=vendor ./* ./.github/* | grep -v ' . basename(__FILE__)));
+if (str_starts_with(strtoupper(PHP_OS), 'WIN')) {
+    $files = preg_split("/\\r\\n|\\r|\\n/", run("dir /S /B * | findstr /v /i .github | findstr /v /i vendor | findstr /v /i ".basename(__FILE__)." | findstr /r /i /M /F:/ \":author :vendor :package VendorName skeleton vendor_name vendor_slug author@domain.com\""));
+}else{
+    $files = explode(PHP_EOL, run('grep -E -r -l -i ":author|:vendor|:package|SIDESO|wa-360dialog|vendor_name|vendor_slug|ctrujillopineda@gmail.com" --exclude-dir=vendor ./* ./.github/* | grep -v ' . basename(__FILE__)));
```

Comment:
Could you refactor this so that we don't use an `else`? But the grep statements in their own functions `replaceForWindows`, `replaceForAllOtherOSes` and use a ternary to choose between them.

Also its seems that `ctrujillopineda@gmail.com` shouldn't be there.